README.md

```diff
@@ -16,8 +16,8 @@ Packages shipped from this repository generally support all the officially
 supported versions of [.NET
 Core](https://dotnet.microsoft.com/download/dotnet-core), and [.NET
 Framework](https://dotnet.microsoft.com/download/dotnet-framework) except for
-`.NET Framework 3.5 SP1`. Any exceptions to this are noted in the individual
-`README.md` files.
+versions lower than `.NET Framework 4.6.1`.
```

Comment:
It seems we don't support `netcoreapp2.1`, too.

Reply:
I see `netstandard2.0` in at least `opentelemetry.1.2.0-alpha1.nupkg` which supports `.NET Core 2.0+`.
See: https://docs.microsoft.com/en-us/dotnet/standard/net-standard

Reply:
If we are calling out .NET Core 2.1, we need to remove that on Aug 22 when .NET Core 2.1 goes out of support (so the original statement "support all the officially supported version of .NET Core" will cover it).